MAUI/Maps/zoom-pan.md

```diff
@@ -13,6 +13,11 @@ It is possible to zoom in and out for any layer to take a closer look at a speci
 
 The procedure for zooming and panning for both layers is very similar.
 
+To learn more about our .NET MAUI Maps zoom and pan, you can check the below video.
```

Comment:
To learn more about the .NET MAUI Maps Zoom and Pan, you can check the following video.